### PR TITLE
Fix beat Service __reduce__ error

### PR DESCRIPTION
**Pull request by AXHMFHNP**
Fix beat Service `__reduce__` error

### CODE REVIEW

**Review by auvipy — COMMENTED** [review]
can you please tell what error you are fixing with this?

**Comment by AXHMFHNP** [comment]
“TypeError: 'Celery' object is not callable” reported at celery beat service start.
```
        debug('beat: Ticking with max interval->%s',
              humanize_seconds(self.scheduler.max_interval))
```
self.scheduler.max_interval -> self.scheduler -> self.get_scheduler() 
```
        return symbol_by_name(self.scheduler_cls, aliases=aliases)(
```
 self.scheduler_cls becomes Celery object due to the inconsistency between the order of `__init__` parameters and the order returned by `__reduce__`

**Comment by AXHMFHNP** [comment]
Done

File: t/unit/test_beat.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class test_Service:
+
+    def test___reduce__(self):
+
+        s1 = Service(mock.sentinel.app,
+                     max_interval=mock.sentinel.max_interval,
+                     schedule_filename=mock.sentinel.schedule_filename,
+                     scheduler_cls=mock.sentinel.scheduler_cls)
+
+        s2 = pickle.loads(pickle.dumps(s1))
```
Comment (auvipy) [inline t/unit/test_beat.py:16]: why specifically pickle? can we also try json here beside pickle?

**Review by auvipy — CHANGES_REQUESTED** [review]

**Comment by AXHMFHNP** [comment]
The problems that occurred with the old version are not reproduced in the new version.

**Pull request closed without merging**